src/pages/help/support.mdx

```diff
@@ -0,0 +1,111 @@
+
+</li>
+<li class="bx--list__item">
+
+If you can’t find an existing answer to your question, please ask the team directly:
+
+
+
+<ol class="bx--list--nested">
+<li class="bx--list__item">
+
+For general questions, please use the [#IBM-design-language](https://ibm-studios.slack.com/messages/C0U35LB2B) Slack channel
+
+</li>
+<li class="bx--list__item">
+
+For designer questions, please use: [#carbon-design-system](https://ibm-studios.slack.com/messages/C0M053VPT/)
+
+</li>
+<li class="bx--list__item">
+
+For developer questions, please use: [#carbon-components](https://ibm-studios.slack.com/messages/C046Y0YUD/), or [#carbon-react](https://ibm-studios.slack.com/messages/C2K6RFJ1G/)
+
+</li>
+</ol>
+</li>
+<li class="bx--list__item">
+
+If you do not have access to IBM’s Slack channels, please email your question to [designlanguage@us.ibm.com]() Messages will be addressed by the IBM Brand Experience Design team as quickly as possible.
+
+</li>
+</ol>
+
+Please note that the IBM Brand Experience Design team is small and working through a backlog of user-initiated features to deliver design elements and guidance. We will get to your question as soon as possible.
+
+## Enablement
+
+If you have explored the [IBM Design](https://www.ibm.com/design/) website, and familiarized yourself with the new [IBM Design Language](https://www.ibm.com/design/language/) website, including the [Design Philosophy](https://www.ibm.com/design/language/philosophy/pov) and [Principles](https://www.ibm.com/design/language/philosophy/principles/), you are ready to learn more:
+
+
+- Start here with [IBM Design Essentials Course](https://w3.ibm.com/design/essentials). These six sessions will cover the must-know essentials, so you can understand, design, and implement with the new IBM Design Ethos, Language, and System. <br/>
+- Explore the [Gallery](https://www.ibm.com/design/language/gallery) of the new IBM Design Language in use.<br/>
+- Access elements and setup the [IBM Design kit](https://www.carbondesignsystem.com/resources/?&_ga=2.192292330.1162213957.1569426322-42838645.1566382348#theme-libraries) <br/>
+- Become a member of the [#ibm-design-kit](https://ibm-studios.slack.com/archives/CDT9TSUAJ) slack channel <br/>
+- Become a member of the [#ibm-design-language](https://ibm-studios.slack.com/messages/C0U35LB2B) channel <br/>
+- Request to become an IBM Design Language sponsor user group by emailing: [designlanguage@us.ibm.com]() with the subject: IDL Sponsor User Group request. Once you are part of the Sponsors User Group, participate in the IBM Design Language [Wake board](https://ibm.wake.com/space/20) <br/>
```

Comment:
link is missing URL

Reply:
It's just the email hyperlink no URL for that.